_includes/includes/ui/keyboard-details.php

```diff
@@ -361,27 +375,32 @@ protected static function WriteTitle() {
     }
 
     protected static function WriteDownloadBoxes() {
-      global $embed, $session_query_q, $embed_win, $embed_version, $pageDevice;
-      global $embed_target;
+      global $embed, $embed_win, $embed_version;
+      global $embed_developer;
 
       // We'll write all the different platforms here and then let Bowser determine
       // which box to show. This is true for both embedded and web-based viewing.
 
       $text = '';
 
-      foreach(self::platformTitles as $platform => $title) {
-        if($platform != 'desktopWeb' && $platform != 'mobileWeb')
-          $text .= self::download_box($platform);
-      }
+      if($embed_developer) {
+        $text .= self::WriteDeveloperCloneBox();
+        echo $text;
+      } else {
+        foreach(self::platformTitles as $platform => $title) {
+          if($platform != 'desktopWeb' && $platform != 'mobileWeb')
+            $text .= self::download_box($platform);
+        }
 
-      $text .= self::download_box('unknown');
+        $text .= self::download_box('unknown');
 
-      if ($embed_win && isset(self::$keyboard->minKeymanVersion) && version_compare(self::$keyboard->minKeymanVersion, $embed_version) > 0) {
+        if ($embed_win && isset(self::$keyboard->minKeymanVersion) && version_compare(self::$keyboard->minKeymanVersion, $embed_version) > 0) {
```

Comment:
`$embed_win` does probably not mean Windows, otherwise I'd wonder why we do that check only for Windows.

Reply:
`$embed_win` does mean Windows. That's a good question, and I suspect it's because we didn't have Keyman version information being passed in to the site from Linux and mac? Perhaps we should open an issue to explore that in a future version?